packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -152,28 +158,34 @@ describe('The order completion handler', () => {
   })
 
   it.each`
-    desc                                                                | licenceFor | postal   | method      | reminder
-    ${'Postal licence for you with none digital confirmation'}          | ${true}    | ${true}  | ${'Letter'} | ${'Text'}
-    ${'Postal licence for someone else with none digital confirmation'} | ${false}   | ${true}  | ${'Letter'} | ${'Text'}
-    ${'Postal licence for you with digital confirmation'}               | ${true}    | ${true}  | ${'Text'}   | ${'Text'}
-    ${'Postal licence for someone else with digital confirmation'}      | ${false}   | ${true}  | ${'Text'}   | ${'Text'}
-    ${'Digital licence for you'}                                        | ${true}    | ${false} | ${'Text'}   | ${'Text'}
-    ${'Digital licence for someone else'}                               | ${false}   | ${false} | ${'Text'}   | ${'Text'}
-    ${'Recurring payment with postal reminder'}                         | ${true}    | ${false} | ${'Text'}   | ${'Letter'}
-    ${'Recurring payment with digital reminder'}                        | ${true}    | ${false} | ${'Text'}   | ${'Text'}
-  `('$desc', async ({ desc, licenceFor, postal, method, reminder }) => {
-    const permission = getSamplePermission({
-      isLicenceForYou: licenceFor,
+    desc                                                                         | licenceFor | postal   | method      | reminder    | length
+    ${'12 month postal licence for you with none digital confirmation'}          | ${true}    | ${true}  | ${'Letter'} | ${'Text'}   | ${'12M'}
+    ${'12 month postal licence for someone else with none digital confirmation'} | ${false}   | ${true}  | ${'Letter'} | ${'Text'}   | ${'12M'}
+    ${'12 month postal licence for you with digital confirmation'}               | ${true}    | ${true}  | ${'Text'}   | ${'Text'}   | ${'12M'}
+    ${'12 month postal licence for someone else with digital confirmation'}      | ${false}   | ${true}  | ${'Text'}   | ${'Text'}   | ${'12M'}
+    ${'12 month digital licence for you'}                                        | ${true}    | ${false} | ${'Text'}   | ${'Text'}   | ${'12M'}
+    ${'12 month digital licence for someone else'}                               | ${false}   | ${false} | ${'Text'}   | ${'Text'}   | ${'12M'}
+    ${'12 month recurring payment with postal reminder'}                         | ${true}    | ${false} | ${'Text'}   | ${'Letter'} | ${'12M'}
+    ${'12 month recurring payment with digital reminder'}                        | ${true}    | ${false} | ${'Text'}   | ${'Text'}   | ${'12M'}
+    ${'8 day licence sets as non_postal'}                                        | ${true}    | ${true}  | ${'Text'}   | ${'Text'}   | ${'8D'}
```

Comment:
Think this description could be a bit clearer – are we testing an 8 day licence where the postalFulfilment is set to true? Would this happen?

Reply:
for 8 day licence postalFulfilment isn't defined